lambdas/ecs-events-to-slack/notify.py

```diff
@@ -22,10 +22,17 @@ def message_formatter(event, region):
     attachment_status = event_detail["attachments"][0]["status"]
 
     env_vars = event_detail["overrides"]["containerOverrides"][0]["environment"]
+    excludes = {}
```

Comment:
This should be an empty list `[]`, instead of a `dict`, since you're making a `list` on line 27.

(If we though the list of env vars was going to be huge, I'd say it should be a `set`, but since this will have like 3 items I don't care)